tests/tests.py

```diff
@@ -466,21 +470,40 @@ class BoringForm(forms.Form):
     boredom = forms.IntegerField()
     excitement = forms.IntegerField()
 
+
 class TemplatetagTests(TestCase):
-    boring_form_html = (
-        u'<fieldset class="fieldset_main">'
-        u'<ul>'
-        u'<li>'
-        u'<label for="id_boredom">Boredom%(suffix)s</label>'
-        u'<input type="%(type)s" name="boredom" id="id_boredom" />'
-        u'</li>'
-        u'<li>'
-        u'<label for="id_excitement">Excitement%(suffix)s</label>'
-        u'<input type="%(type)s" name="excitement" id="id_excitement" />'
-        u'</li>'
-        u'</ul>'
-        u'</fieldset>'
-        ) % {'type': number_field_type, 'suffix': label_suffix}
+    @property
```

Comment:
`required` change how it works in Django 1.11. These to field converted to properties makes it work in both django versions.